File: kotlin/multiples-of-3-or-5/solution.kt
// https://www.codewars.com/kata/514b92a657cdc65150000006

fun solution(upTo: Int): Int {
    var sum = 0
    for (n in 0..upTo-1) {
        if (n % 3 == 0 || n % 5 == 0) {
            sum += n
        }
    }
    return sum
}

File: lua/multiples-of-3-or-5/solution.lua
-- https://www.codewars.com/kata/514b92a657cdc65150000006

local solution = {}

function solution.solution(upTo)
  local sum = 0
  for n = 0, upTo - 1 do
    if n % 3 == 0 or n % 5 == 0 then
      sum = sum + n
    end
  end
  return sum
end

return solution

